agir/events/serializers.py

```diff
@@ -253,6 +260,30 @@ def get_groups(self, obj):
             ],
         ).data
 
+    def get_participants(self, obj):
+        return None
+
+    def get_organizers(self, obj):
+        return None
+
+
+class EventAdvancedSerializer(EventSerializer):
+    def get_participants(self, obj):
+        return PersonSerializer(
+            obj.attendees,
+            context=self.context,
+            many=True,
+            fields=["id", "email", "firstName", "lastName", "displayName"],
+        ).data
+
+    def get_organizers(self, obj):
+        return PersonSerializer(
+            obj.organizers,
+            context=self.context,
+            many=True,
+            fields=["id", "email", "firstName", "lastName", "displayName"],
+        ).data
```

Comment:
Si tu as besoin uniquement de ces informations, c'est mieux pour les performances de ne pas utiliser le PersonSerializer, mais de renvoyer directement des dictionnaires :
```suggestion
def get_participants(self, obj):
    return [
        {
            "id": person.id,
            "email": person.email,
            "firstName": person.first_name,
            "lastName": person.last_name,
            "displayName": person.display_name
        }
        for person in obj.attendees
    ]

def get_organizers(self, obj):
    return [
        {
            "id": person.id,
            "email": person.email,
            "firstName": person.first_name,
            "lastName": person.last_name,
            "displayName": person.display_name
        }
        for person in obj.organizers
    ]
```
Et d'ailleurs tu es sûr d'avoir besoin aussi de firstname et lastname ?